README.md

```diff
@@ -195,9 +199,12 @@ Performs a collection against the target host. A typical collection will general
 - get_adapter_definition():
 Defines the object types and attribute types (metric/property) that are present
 in a collection. Setting these object types and attribute types helps VMware Aria Operations to
-validate, process, and display the data correctly.
+validate, process, and display the data correctly. More importantly, in most cases, it defines the
+Adapter Instance configuration parameters/credentials.
```

Comment:
If it's more important it should be the first thing we talk about.

'in most cases' -> if we include this phrase, there should be some discussion about what the different cases are and why they may be relevant. 

There should be at least a sentence or two about configuration parameters and credentials and how they're used in Aria Ops